tools/ovc/openvino/tools/ovc/logger.py

```diff
@@ -62,7 +62,10 @@ def filter(self, record: log.LogRecord):
         return True  # if regex wasn't set print all logs
 
 
-def init_logger(lvl: str, verbose: bool):
+def init_logger(lvl: str, verbose: bool, python_api_used: bool):
+    if verbose and python_api_used:
+        # We need to not override logger in case of verbose=True to allow user set a log level
```

Comment:
how can user set logger level? In help, there is no info about it.

Reply:
Added info to `convert_model()` docstring.